lumicks/pylake/channel.py

```diff
@@ -121,7 +121,7 @@ def downsampled_over(self, range_list, reduce=np.mean, where='center'):
         if where != 'center' and where != 'left':
             raise ValueError("Invalid argument for where. Valid options are center and left")
 
-        t = np.zeros(len(range_list))
+        t = np.zeros(len(range_list), dtype=np.int64)
```

Comment:
This might make it more obvious that the aim is to match the timestamp `dtype`:
```suggestion
        t = np.zeros(len(range_list), dtype=self._src.timestamps.dtype)
```

Reply:
Good point, thanks!